src/kibana/components/agg_types/_agg_type.js

```diff
@@ -132,6 +132,9 @@ define(function (require) {
      * @return {FieldFromat}
      */
     AggType.prototype.getFormat = function (agg) {
+      if (agg.type && agg.type.dslName === 'range') {
```

Comment:
Would you mind using this pattern instead? https://github.com/elastic/kibana/blob/38adb3f5c909c744a7e742fe325006e62a42bbc4/src/kibana/components/agg_types/buckets/date_range.js#L25-L27